src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
```

Comment:
Please add the `toString` method. Otherwise, you'll print out the memory references, not the actual `Manufacturer` data. Also, it's a good practice to have `equals` and `hashcode` methods as well. These two are not required here, but the `toString` one will definitely help.